src/parametrize.js

```diff
@@ -54,9 +54,9 @@ const Parametrize = React.createClass({
     });
 
     return (
-      <div>
+      <span>
```

Comment:
@rawls238 this should definitely be a span. We don't want to wrap our children in a block level element because it will break inline experimental content.